Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
+        },
+        copy: {
+            // Copy external front end dependencies into appropriate directories
+            frontEndDependencies: {
+                files: [
+                    // D3
+                    {expand: true, cwd: "./node_modules/d3/", src: "**", dest: ".//src/lib/ext/d3/"},
```

Comment:
Why do d3 and flocking libraries are placed in "src/lib/ext/" instead of "src/lib/"?

BTW, is it a typo to have double slashes for ".//src..."?